src/liballoc/rc.rs

```diff
@@ -10,7 +10,8 @@
 
 #![allow(deprecated)]
 
-//! Single-threaded reference-counting pointers.
+//! Single-threaded reference-counting pointers. "Rc" stands for "Reference
```

Comment:
When I compare "atomically reference-counted" with "atomic reference counter," the tech-jargon-ness of the former seems a bit stronger than that of the latter. Just my personal impression.